src/Bicep.Core/Syntax/TypeDeclarationSyntax.cs

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using Bicep.Core.Navigation;
+using Bicep.Core.Parsing;
+
+namespace Bicep.Core.Syntax;
+
+public class TypeDeclarationSyntax : StatementSyntax, ITopLevelNamedDeclarationSyntax
+{
+    public TypeDeclarationSyntax(Token keyword, IdentifierSyntax name, SyntaxBase assignment, SyntaxBase value)
```

Comment:
Is this constructor used?

Reply:
Nope, I can remove it. I think this is leftover from some earlier tests before I moved everything into baselined tests.